distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/JobManagerPaths.java

```diff
@@ -3,88 +3,100 @@
+                String.format("work-share-%s.lock", workItem));
     }
 
-    public String getWorkersAlivePath() {
-        return ZKPaths.makePath(rootPath, "alive");
+    String getRegistrationVersion() {
+        return path(REGISTRATION_VERSION);
     }
 
-    String getAvailableWorkPooledJobPath(String workerId) {
-        return ZKPaths.makePath(getWorkerPath(workerId), "available", WORK_POOLED_JOB_ID);
+    String getWorkersPath() {
+        return path(WORKERS);
     }
 
-    String getAvailableWorkPoolPath(String workerId, String jobId) {
-        return ZKPaths.makePath(getAvailableWorkPooledJobPath(workerId), jobId, WORK_POOL);
+    String getWorkerPath(String workerId) {
+        return path(WORKERS, workerId);
     }
 
-    String getAssignedWorkPoolPath(String workerId, String jobId) {
-        return ZKPaths.makePath(getAssignedWorkPooledJobsPath(workerId), jobId, WORK_POOL);
+    String getWorkerAssignedJobsPath(String workerId) {
+        return path(WORKERS, workerId, ASSIGNED);
     }
 
     String getAssignedWorkPooledJobsPath(String workerId) {
-        return ZKPaths.makePath(getWorkersPath(), workerId, "assigned", WORK_POOLED_JOB_ID);
+        return path(WORKERS, workerId, ASSIGNED, WORK_POOLED_JOB_ID);
     }
 
     String getAssignedWorkPooledJobsPath(String workerId, String jobId) {
-        return ZKPaths.makePath(getAssignedWorkPooledJobsPath(workerId), jobId);
+        return path(WORKERS, workerId, ASSIGNED, WORK_POOLED_JOB_ID, jobId);
     }
 
-    String getAvailableWorkItemPath(String workerId, String jobId, String workItemId) {
-        return ZKPaths.makePath(getAvailableWorkPoolPath(workerId, jobId), workItemId);
+    String getAssignedWorkPoolPath(String workerId, String jobId) {
+        return path(WORKERS, workerId, ASSIGNED, WORK_POOLED_JOB_ID, jobId, WORK_POOL);
     }
 
     String getAssignedWorkItemPath(String workerId, String jobId, String workItemId) {
-        return ZKPaths.makePath(getAssignedWorkPoolPath(workerId, jobId), workItemId);
+        return path(WORKERS, workerId, ASSIGNED, WORK_POOLED_JOB_ID, jobId, WORK_POOL, workItemId);
     }
 
-    String getWorkPooledLocksPath() {
-        return ZKPaths.makePath(rootPath, "locks", WORK_POOLED_JOB_ID);
+    String getWorkerAvailableJobsPath(String workerId) {
+        return path(WORKERS, workerId, AVAILABLE);
     }
 
-    String getWorkItemLock(String jobId, String workItem) {
-        return ZKPaths.makePath(rootPath, "locks", WORK_POOLED_JOB_ID, jobId,
-                String.format("work-share-%s.lock", workItem));
+    String getAvailableWorkPooledJobPath(String workerId) {
+        return path(WORKERS, workerId, AVAILABLE, WORK_POOLED_JOB_ID);
+    }
+
+    String getAvailableWorkPoolPath(String workerId, String jobId) {
+        return path(WORKERS, workerId, AVAILABLE, WORK_POOLED_JOB_ID, jobId, WORK_POOL);
     }
 
+    String getAvailableWorkItemPath(String workerId, String jobId, String workItemId) {
```

Comment:
unused method, but i used it in next branch: https://github.com/ru-fix/distributed-job-manager/pull/25